### PR TITLE
Update AboutUs.md

### PR DESCRIPTION
**Pull request by KSSWSept20**

### CODE REVIEW

File: docs/AboutUs.md [hunk 1]
```diff
@@ -5,57 +5,35 @@
 
 # About Us
 
-We are a team based in the [School of Computing, National University of Singapore](http://www.comp.nus.edu.sg).
-
-You can reach us at the email `seer[at]comp.nus.edu.sg`
+We are a team taking the Software Engineering (CS2310T) course offered by [School of Computing, National University of Singapore](http://www.comp.nus.edu.sg).
```
Comment (ziiqii) [inline docs/AboutUs.md:8]: Might want to fix the typo CS2310T to CS2103T

File: docs/AboutUs.md [hunk 2]
```diff
@@ -5,57 +5,35 @@
...
-* Role: Developer
-* Responsibilities: Data
+<img src="images/zhuuyicheng.png" width="200px">
 
-### Jean Doe
+email address: `e0957889@u.nus.edu`<br>
+[[github](https://github.com/zhuuyicheng)]
 
-<img src="images/johndoe.png" width="200px">
+### Xu Zi Qi
```
Comment (ziiqii) [inline docs/AboutUs.md:26]: Could you help to remove the spaces for Yicheng and me (Ziqi)?

**Review by ziiqii — COMMENTED** [review]
LGTM, just a few typos to fix!

File: docs/AboutUs.md (hunk 2 above)
Comment (KSSWSept20) [reply docs/AboutUs.md]: ok, done.

File: docs/AboutUs.md (hunk 1 above)
Comment (KSSWSept20) [reply docs/AboutUs.md]: thanks, changed.

**Review by ziiqii — APPROVED** [review]
LGTM!

**Review by Tanzhiheng26 — APPROVED** [review]
LGTM

**Pull request merged**